ci/build-docker.sh

```diff
@@ -22,10 +22,23 @@ set -eu
 mkdir -p gccpp/build-output
 cd gccpp/build-output
 
-cmake -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" ${CMAKE_FLAGS:-} ..
+CMAKE_COMMAND="cmake"
+if [ "${SCAN_BUILD}" = "yes" ]; then
+    CMAKE_COMMAND="scan-build cmake"
+fi
+${CMAKE_COMMAND} -DCMAKE_BUILD_TYPE="${BUILD_TYPE}" ${CMAKE_FLAGS:-} ..
+
+# If scan-build is enabled we need to manually compile the dependencies,
+# otherwise the static analyzer finds issues in them, and there is no way to
+# ignore them.
+if [ "${SCAN_BUILD}" = "yes" ]; then
+    make -j ${NCPU} -C bigtable googleapis
```

Comment:
It's 2-space indent elsewhere in the file (below).

Reply:
Done.